packages/react-components-lab/package.json

```diff
@@ -2,21 +2,21 @@
   "version": "0.2.11",
   "license": "MIT",
   "main": "dist/index.js",
-  "name": "@dhi-gras/react-components-lab-test",
+  "name": "@dhi/water-warriors-lab",
```

Comment:
I think I personally prefer the old one, matches the core package and is more neutral

Reply:
yes I agree, I have to change those

Reply:
I agree, let us rather stick to the conservative / boring naming for now. But let's keep the warrior vibe...